### PR TITLE
[FUSEQE-4294] Add SNS test

### PR DESCRIPTION
**Pull request by avano**
//test: `@sqs or @sns`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/utils/SQSUtils.java [hunk 1]
```diff
@@ -102,14 +104,24 @@ public void sendMessages(String queueName, List<String> messageContents) {
...
-     * Returns all messages.
+     * Returns all messages. You can get maximum of 10 messages per poll and even then the count of messages returned
+     * seems somewhat random, so repeat until all messages are received
      *
      * @return messages list
      */
     public List<Message> getMessages(String queueName) {
-        return client.receiveMessage(b -> b.queueUrl(getQueueUrl(queueName))
+        final int queueSize = getQueueSize(queueName);
+        List<Message> allMessages = new ArrayList<>(client.receiveMessage(b -> b.queueUrl(getQueueUrl(queueName))
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/utils/SQSUtils.java]: Two things here 
1) the indenting is kind of confusing
2) the list initialization is exactly the same as the inside of the loop below, could be removed here and just let the loop handle it.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**